### PR TITLE
ref(crons): Add model field comments

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: src/sentry/monitors/models.py [hunk 1]
```diff
@@ -387,20 +387,54 @@ class MonitorCheckIn(Model):
...
+    """
     duration = BoundedPositiveIntegerField(null=True)
+    """
+    The total number in milliseconds that the check-in took to execute. This is
+    generally computed from the difference between the open and close check-in.
+    """
     date_added = models.DateTimeField(default=timezone.now, db_index=True)
+    """
+    Represents the time the checkin was made. This CAN BE back-dated in some
+    cases, and does not necissarily represent the insertion time of the row in
```
Comment (rjo100) [inline src/sentry/monitors/models.py:409]:
```suggestion
    cases, and does not necessarily represent the insertion time of the row in
```

File: src/sentry/monitors/models.py [hunk 2]
```diff
@@ -387,20 +387,54 @@ class MonitorCheckIn(Model):
...
+        db_index=True,
     )
-    config = JSONField(default=dict)
+    """
+    The status of the check-in
+    """
     duration = BoundedPositiveIntegerField(null=True)
+    """
+    The total number in milliseconds that the check-in took to execute. This is
+    generally computed from the difference between the open and close check-in.
```
Comment (rjo100) [inline src/sentry/monitors/models.py:404]:
```suggestion
    generally computed from the difference between the opening and closing check-ins.
```

File: src/sentry/monitors/models.py [hunk 3]
```diff
@@ -387,20 +387,54 @@ class MonitorCheckIn(Model):
...
+    """
+    Represents the time the checkin was made. This CAN BE back-dated in some
+    cases, and does not necissarily represent the insertion time of the row in
+    the database.
+    """
     date_updated = models.DateTimeField(default=timezone.now)
-    attachment_id = BoundedBigIntegerField(null=True)
-    # Holds the time we expected to receive this check-in without factoring in margin
+    """
+    Last update time for the check-in. Can be used to extend the lifetime of a
```
Comment (rjo100) [inline src/sentry/monitors/models.py:414]: this isn't true. this is the last time we received an `IN_PROGRESS` status for the check-in. the second sentence is true

File: src/sentry/monitors/models.py [hunk 4]
```diff
@@ -474,13 +508,28 @@ class MonitorEnvironment(Model):
...
+    list, since it would be possible to determine this by looking at recent
+    check-ins. It is denormalized for simplicity.
+    """
+    next_checkin = models.DateTimeField(null=True)
+    """
+    The expected time that the next-checkin will occur
+    """
+    next_checkin_latest = models.DateTimeField(null=True)
+    """
+    The latest expected time that the next-checkin will occur. This is computed
```
Comment (rjo100) [inline src/sentry/monitors/models.py:525]:
```suggestion
    The latest expected time that the next-checkin can occur without generating a missed check-in. This is computed
```

File: src/sentry/monitors/models.py [hunk 5]
```diff
@@ -474,13 +508,28 @@ class MonitorEnvironment(Model):
...
+    using the user-configured margin for the monitor.
+    """
     last_checkin = models.DateTimeField(null=True)
+    """
+    Time of the most recent user-check in. This does not include auto-generated
```
Comment (rjo100) [inline src/sentry/monitors/models.py:530]:
```suggestion
    Time of the start of the most recent user-check in. This does not include auto-generated
```

Comment (evanpurkhiser) [reply src/sentry/monitors/models.py:530]: actually is it EXPECTED time of the most recent check in?

Comment (rjo100) [reply src/sentry/monitors/models.py:530]: no we update it on every `mark_failed` or `mark_ok`

Comment (evanpurkhiser) [reply src/sentry/monitors/models.py:530]:
```suggestion
    date_added time of the most recent user-check in. This does not include auto-generated
```

How about this?

Comment (rjo100) [reply src/sentry/monitors/models.py:530]: yeah that works

File: src/sentry/monitors/models.py (hunk 3 above)
Comment (evanpurkhiser) [reply src/sentry/monitors/models.py:419]: oh do we not update this when we close a check-in either?

**Review by rjo100 — APPROVED** [review]

**Pull request merged**